system-int-test/test/cmr/system_int_test/search/collection_with_new_granule_search_test.clj

```diff
@@ -193,8 +193,12 @@
       (testing "has_granules_created_at parameter by itself"
         (util/are3
           [date-ranges expected-results]
-          (let [actual-results (search/find-refs :collection {:has-granules-created-at date-ranges})]
-            (d/assert-refs-match expected-results actual-results))
+          (let [params {:has-granules-created-at date-ranges}
+                json-search-results (search/find-concepts-json :collection params)
+                actual-results (search/find-refs :collection params)]
+            (d/assert-refs-match expected-results actual-results)
+            (is (= (set (map :concept-id expected-results))
```

Comment:
Since the issue seems to be related to the requested output format we should add a test for each supported output format.

Each test could be wrapped in a `(testing "has_granules_created_at in JSON format" ... )`.